ovrstat/player_stats.go

```diff
@@ -120,11 +121,15 @@ func playerStats(profilePath string, platform string) (*PlayerStats, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer res.Body.Close()
+
+	// Decode received JSON
 	if err := json.NewDecoder(res.Body).Decode(&platforms); err != nil {
 		return nil, ErrPlayerNotFound
 	}
 
+	// Close it again
+	res.Body.Close()
```

Comment:
Same here, the body's need to be defer closed before they are used. If you're afraid of rewriting `res` we could perhaps rename the response variable to something other than `res` and defer close that instead.

Reply:
@s32x yeah, sounds like a solution. Will send fix a little bit later.